etg/window.py

```diff
@@ -136,6 +136,16 @@ def run():
     m1.find('externalLeading').out = True
 
     c.find('GetHandle').type = 'wxUIntPtr*'
+    c.find('GetHandle').detailedDoc = [
```

Comment:
I don't know what the right API to do this is, and I couldn't get inline markup working right.